src/theme/SearchBar/index.js

```diff
@@ -1,8 +1,23 @@
-import React from "react";
+import React, { useEffect } from "react";
 import SearchBar from "@theme-original/SearchBar";
 import AskAI from "@site/components/AskAI";
+import { useColorMode } from "@docusaurus/theme-common";
 
+const useRadixTheme = () => {
+  const { colorMode } = useColorMode();
+
+  //Radix color variables require a class on the root element
+  useEffect(() => {
+    //Add classes for radix color variables
+    if (colorMode === "dark") {
+      document.documentElement.classList.add("dark-theme");
+    } else {
+      document.documentElement.classList.remove("dark-theme");
+    }
+  }, [colorMode]);
+};
```

Comment:
It's a hack. Please add documentation on why it's here and how it works. I'm afraid that it will stop working one day and no one will be able to understand how theming ever worked.

Reply:
This hack was removed in later commit